bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/visitors/IdentifyDroppedPersistentTableVisitor.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ package com.google.zetasql.toolkit.antipattern.parser.visitors;
+
+import com.google.zetasql.parser.ASTNodes.ASTCreateTableStatement;
+import com.google.zetasql.parser.ASTNodes.ASTDropStatement;
+import com.google.zetasql.parser.ParseTreeVisitor;
+ import com.google.zetasql.toolkit.antipattern.AntiPatternVisitor;
+ import com.google.zetasql.toolkit.antipattern.util.ZetaSQLStringParsingHelper;
+ import java.util.ArrayList;
+ import java.util.HashMap;
+ import java.util.Map;
+ import java.util.stream.Collectors;
+
+ public class IdentifyDroppedPersistentTableVisitor extends ParseTreeVisitor
+     implements AntiPatternVisitor {
+
+   public static final String NAME = "ConvertTableToTemp";
+   private final String CONVERT_TO_TEMP_SUGGESTION_MESSAGE =
+       "Persistent table dropped without TEMP: Table %s defined at line %d is dropped. Consider converting to temporary.";
```

Comment:
```suggestion
       "Persistent table dropped: Table %s defined at line %d is dropped. Consider converting to temporary.";
```

If it is persistent then it is implicit that it lacks temp